### PR TITLE
Group scss imports

### PR DESCRIPTION
**Pull request by Dietr**
See #194

### CODE REVIEW

File: app/styles/ember-appuniversum.scss [hunk 1]
```diff
@@ -15,100 +15,16 @@
...
-@import "appuniversum/u-font-family";
-@import "appuniversum/u-font-weights";
-@import "appuniversum/u-max-widths";
-@import "appuniversum/u-paragraphs";
-@import "appuniversum/u-print";
-@import "appuniversum/u-responsive-spacings";
-@import "appuniversum/u-spacings";
-@import "appuniversum/u-visible";
-@import "appuniversum/u-widths";
+@import "ember-appuniversum/a-settings";
```
Comment (Windvis) [inline app/styles/ember-appuniversum.scss:23]: Silly question, but what system are we using for the file naming? What does the `a-` prefix mean here?

File: stories/1-outline/Getting-started.stories.mdx [hunk 2]
```diff
@@ -72,8 +72,19 @@ Then load the SCSS in your `app.scss` file:
 @import "ember-appuniversum";
 ```
 
+If you want more control over the order that SCSS partials are rendered you can import them as groups.
+``` js
```
Comment (Windvis) [inline stories/1-outline/Getting-started.stories.mdx:76]:
```suggestion
``` scss
```

**Review by Windvis — APPROVED** [review]
Nice! Seems to work just as expected when I tried it in loket. :+1:

File: app/styles/ember-appuniversum.scss (hunk 1 above)
Comment (Dietr) [reply app/styles/ember-appuniversum.scss:23]: It's based on ITCSS (https://appuniversum.github.io/appuniversum/guidelines/architecture/). The reason of the prefixes is to easily spot files and their purpose (also having them grouped alphabetically is quite handy). I used the "a" here for "all". "g" (group) was already taken by generic.

**Pull request merged**